specification/communication/Communication.Messages/examples/2024-02-01/Stream_GetStream.json

```diff
@@ -0,0 +1,15 @@
+{
+  "title": "Download a media",
+  "operationId": "StreamOperations_GetStream",
+  "parameters": {
+    "api-version": "2024-02-01",
+    "content-type": "application/json",
+    "id": "d19e68ec-bdd6-4a50-8dfb-cbb1642df6ab",
+    "endpoint": "https://my-resource.communication.azure.com"
+  },
+  "responses": {
+    "200": {
+      "body": {}
```

Comment:
Can we change this to some example data? Maybe also add the octet-stream response header